qt/scientific_interfaces/ISISReflectometry/GUI/Runs/RunsPresenter.cpp

```diff
@@ -447,11 +447,12 @@ IAlgorithm_sptr RunsPresenter::setupLiveDataMonitorAlgorithm() {
   alg->setLogging(false);
   auto const instrument = m_view->getSearchInstrument();
   auto const inputWorkspace = "TOF_live";
+  auto const updateInterval = m_view->getLiveDataUpdateInterval();
   alg->setProperty("Instrument", instrument);
   alg->setProperty("OutputWorkspace", "IvsQ_binned_live");
   alg->setProperty("AccumulationWorkspace", inputWorkspace);
   alg->setProperty("AccumulationMethod", "Replace");
-  alg->setProperty("UpdateEvery", "20");
+  alg->setProperty("UpdateEvery", updateInterval);
```

Comment:
The tests are crashing on this line because the algorithm actually expects a double, not an int! This fixes it for me:

```suggestion
  alg->setProperty("UpdateEvery", static_cast<double>(updateInterval));
```

Reply:
Makes sense, good spot!